be/src/util/proto_util.h

```diff
@@ -0,0 +1,52 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+namespace doris {
+
+const int64_t MAX_REQUEST_BYTE_SIZE = 64 * 1024 * 1024;
+
+inline void row_batch_transfer_attachment(doris::PRowBatch* row_batch, butil::IOBuf* attachment) {
+    row_batch->set_tuple_data_size(row_batch->tuple_data().size());
+    attachment->append(row_batch->tuple_data());
+    row_batch->clear_tuple_data();
+    row_batch->set_tuple_data("");
+}
+
+template <typename Params, typename Closure>
+inline void request_transfer_attachment(Params* brpc_request, Closure* closure) {
```

Comment:
Why not use `row_batch_transfer_attachment()` directly

Reply:
done, combines two methods